package/MDAnalysis/analysis/contacts.py

```diff
@@ -63,6 +63,7 @@
     from MDAnalysis.analysis import contacts
     from MDAnalysis.tests.datafiles import PSF,DCD
     import matplotlib.pyplot as plt
+    import numpy as np
```

Comment:
The example currently fails because of Issue #2191. Thus, in the meantime, you need to import `matplotlib.pyplot` *before* importing `MDAnalysis.tests.datafiles` for it to work. To keep the MDAnalysis imports together, I'd suggest the following order:
```python
import numpy as np
import matplotlib.pyplot as plt
import MDAnalysis as mda
from MDAnalysis.analysis import contacts
from MDAnalysis.tests.datafiles import PSF,DCD
```